### PR TITLE
New badmin fusion testing canisters

### PR DESCRIPTION
**Pull request by ghost**
### Intent of your Pull Request

The current set of fusion test canisters are actually really bad at representing normal in-game fusion parameters. These two new canisters should make it easier for admins to do gimmicks or just mess around on centcomm. 

Inspired by that time Wej and I were attempting to reproduce bugs at centcomm but couldn't because the default test canisters couldn't get the job done.

#### Changelog

:cl:  
rscadd: Added new badmin fusion canisters.
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
Can confirm, admin fusion cans were too weak for Colton

File: yogstation/code/modules/atmospherics/machinery/portable/canister.dm [hunk 1]
```diff
@@ -19,3 +19,24 @@
...
+	air_contents.set_moles(/datum/gas/tritium, 1000)
+	air_contents.set_moles(/datum/gas/plasma, 4500)
+	air_contents.set_moles(/datum/gas/carbon_dioxide, 1500)
+	air_contents.set_temperature(100000)
+
+/obj/machinery/portable_atmospherics/canister/fusion_test_4
+	name = "Cold Fusion Test Canister"	
+	desc = "This should never be spawned in game. Contains dilithium for cold fusion."	
+	icon_state = "green"	
+/obj/machinery/portable_atmospherics/canister/fusion_test_3/create_gas()
```
Comment (swissloaf) [inline yogstation/code/modules/atmospherics/machinery/portable/canister.dm]: This needs to be called fusion_test_4 not fustion_test_3

**Review by swissloaf — CHANGES_REQUESTED** [review]

File: yogstation/code/modules/atmospherics/machinery/portable/canister.dm (hunk 1 above)
Comment (ghost) [reply yogstation/code/modules/atmospherics/machinery/portable/canister.dm]: lmao I missed that one

Comment (ghost) [reply yogstation/code/modules/atmospherics/machinery/portable/canister.dm]: Done

**Review by Amelia0010 — APPROVED** [review]

**Review by alexkar598 — DISMISSED** [review]
Please document the objects in this file

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**